addons/l10n_mx/data/account_tag_data.xml

```diff
@@ -2423,13 +2423,13 @@
                 <field name='name'>503 Devoluciones, descuentos o bonificaciones sobre compras</field>
                 <field name='color'>1</field>
                 <field name='applicability'>accounts</field>
-                <field name='nature'>D</field>
+                <field name='nature'>A</field>
             </record>
             <record id='account_tag_503_01' model='account.account.tag'>
                 <field name='name'>503.01 Devoluciones, descuentos o bonificaciones sobre compras</field>
                 <field name='color'>4</field>
                 <field name='applicability'>accounts</field>
-                <field name='nature'>A</field>
+                <field name='nature'>D</field>
```

Comment:
This is not consistent with the previous code:
https://github.com/Vauxoo/odoo/pull/298/files#diff-35c185081edb70a60723a879fa422faeL2426

It shall not be changed. it must be kept as A.

Reply:
clarification for me: 

As I understand "503.01 Devoluciones, descuentos o bonificaciones sobre compras" is (if can I say) the 'main' account and, for example, "503.01" is one of its childs, so if 503 (the main account) has a 'Debit' nature its child should not have another nature.... am I right?